components/Head.js

```diff
@@ -14,7 +14,7 @@ export default function HeadComponent({
   let type = 'website'
   if (path.startsWith('/docs') || path.startsWith('/blog')) type = 'article'
   if (!image.startsWith('http') && !image.startsWith('https')) image = `${url}${image}`
-  title = `${title} | AsyncAPI Initiative`
+  title = `AsyncAPI Initiative for Event-driven APIs`
```

Comment:
This title is weird. Do we really need this?

Reply:
Also, you removed the dynamic title. Why? This only will make it worst for SEO.

Reply:
ah, you are right, I brought it back and just changed that main page doesn't have `Home | `

I suggest we give it a try as it won't cost us much, and I will check in a month if it got worst or better or the world did not notice anything